api/app/tests/apps/api/address/test_validation.py

```diff
@@ -7,17 +7,86 @@
 from requests.exceptions import ConnectionError
 from requests_mock.mocker import Mocker
 
-from signals.apps.api.v1.validation import (
-    AddressValidation,
+from signals.apps.api.v1.validation.address.bag import AddressValidation
+from signals.apps.api.v1.validation.address.base import (
     AddressValidationUnavailableException,
     NoResultsException
 )
+from signals.apps.api.v1.validation.address.pdok import PDOKAddressValidation
+
+
+class PDOKTestAddressValidation(SimpleTestCase):
```

Comment:
nit: generally we name test classes `Test<something>`

no blocker, but adressing this would be a bit more consistent

Reply:
Changed